internal/data/extension_pair.go

```diff
@@ -64,7 +65,7 @@ func (rp extensionPairRepo) DeleteExtensionPairs(ctx context.Context, blockNumbe
 	return nil
 }
 
-func (rp extensionPairRepo) ParseExtensionPairs(blockNumber uint64, entry biz.Entry) (pairs []biz.ExtensionPair, err error) {
+func (rp extensionPairRepo) ParseExtensionPairs(blockNumber uint64, entry biz.Entry) (pairs []biz.ExtensionPair, subKeyPairs []biz.SubKeyPair, err error) {
```

Comment:
Considering that extension is either subkey or social recovery(there may be more types in the future), it is suggested that the data structure of the subkey can be directly put into extension, and extension judges whether it is subkey or social recovery by `sub_type`. What do you think?

Reply:
I see what you mean, it's a better way